docs/api/storage.yml

```diff
@@ -91,6 +91,15 @@ paths:
           required: false
           type: string
 
+        - in: formData
+          name: labels
+          description: Optional labels
+          required: false
+          type: array
+          items:
+            type: string
+          collectionFormat: pipes
```

Comment:
Could we change this to comma separated. Seems more common to me.

Reply:
done